concepts/units-of-measurement.md

```diff
@@ -133,7 +133,9 @@ SI:
 | ElectricPotential      | Volt                             | V      |
 | ElectricCapacitance    | Farad                            | F      |
 | ElectricCharge         | Coulomb                          | C      |
+| ElectricCharge         | Ampere Hour                      | A h    |
```

Comment:
Analog to the implementation also in the documentation the correction without spaces.